src/main/java/com/gtnewhorizons/navigator/api/model/steps/InteractableStep.java

```diff
@@ -10,6 +10,8 @@ public interface InteractableStep extends RenderStep {
 
     void onActionKeyPressed();
 
+    boolean onActionWithKeyPressed(int keyCode);
```

Comment:
This needs to be a default method or all the dependents will break, I also think `onKeyPressed` is a more fitting name for this method.
You can change it to something like the following which means you can also remove all the references to `onActionKeyPressed` from the various LayerRenderers. If you do this then `onActionKeyPressed` should be turned into an empty default method.

```suggestion
       default boolean onKeyPressed(int keyCode) {
        if (Util.isKeyPressed(NavigatorApi.ACTION_KEY)) {
            onActionKeyPressed();
            return true;
        }

        return false;
    }
```

Reply:
Fixed